composer.json

```diff
@@ -16,21 +16,18 @@
             "homepage": "http://www.aoe.com"
         }
     ],
-    "config": {
-        "secure-http": false
-    },
     "replace": {
         "happy_feet": "*"
     },
     "require": {
         "php": ">=5.3.0",
```

Comment:
As you've dropped PHP <5.6 from the TravisCI build matrix, I'd bump the requirements, too. Otherwise we would theoretically support PHP versions not covered by automated tests.

Reply:
Done